test/http-test.js

```diff
@@ -454,6 +454,50 @@ describe('HTTP', function() {
     assert.strictEqual(blocks.length, 10);
   });
 
+  // depends on the previous test to generate blocks
+  it('should fetch block header', async () => {
+    // fetch corresponding header and block
+    const header = await nclient.get(`/header/${7}`);
```

Comment:
Why `/header/${7}`, instead of `/header/7`? Also, it doesn't look like a hash is every used, in the tests, and only the height.

Reply:
Moved `7` to be a variable so that it can easily be changed in the future if the number of blocks being mined in the previous test is made to be less.